testlib/src/test/java/io/spine/testing/TestsShould.java

```diff
@@ -136,6 +139,65 @@ void whenFalse() {
         }
     }
 
+    @Nested
+    @DisplayName("Assert matches mask")
+    class AssertMatchesMask {
+
+        private Timestamp timestampMsg;
+
+        @BeforeEach
+        void setUp(){
+            long currentTime = Instant.now()
+                                      .toEpochMilli();
+            timestampMsg = Timestamp.newBuilder()
+                                    .setSeconds(currentTime)
+                                    .build();
+        }
+
+        @Test
+        @DisplayName("when field is matched")
+        void fieldIsPresent() {
+            String fieldPath = Timestamp.getDescriptor()
+                                        .getFields()
+                                        .get(0)
+                                        .getFullName();
+            FieldMask.Builder builder = FieldMask.newBuilder();
+            builder.addPaths(fieldPath);
+            FieldMask fieldMask = builder.build();
+
+            assertMatchesMask(timestampMsg, fieldMask);
+        }
+
+        @Test
+        @DisplayName("throws the exception when field is not present")
+        void fieldIsNotPresent() {
+            String fieldPath = Any.getDescriptor()
+                                  .getFields()
+                                  .get(0)
+                                  .getFullName();
+            FieldMask.Builder builder = FieldMask.newBuilder();
+            builder.addPaths(fieldPath);
+            FieldMask fieldMask = builder.build();
+
+            assertThrows(AssertionError.class, () -> assertMatchesMask(timestampMsg, fieldMask));
+        }
+
+        @Test
+        @DisplayName("throws the exception when field is not set")
```

Comment:
`AssertionError` is not an exception. Same above.

Also, I think `...when the field value is not set`.